pyfar/constants/constants.py

```diff
@@ -433,3 +434,149 @@ def density_of_air(
 
     # Equation 6.71
     return P / (R_a * (temperature + 273.15)) * (1+C)/(1+alpha*C)
+
+def fractional_octave_frequencies_nominal(num_fractions=1,
+                                          frequency_range=(20, 20e3)):
+    """Return the nominal center frequencies for octave-band and
+    one-third-octave-band filters.
+
+    Nominal center frequencies, as specified in the IEC 61260-1:2014 standard
+    [#]_, are standardized values that approximate the exact center
```

Comment:
```suggestion
    Nominal center frequencies, as specified in the IEC 61672-1 standard
    [#]_, are standardized values that approximate the exact center
```
i think here we should ref to the broader range (10-20)